### ISSUE TITLE
Update config example in the Readme to the lates necessary version

### ISSUE THREAD

**Issue by cheburan** [MRC-Epid-it24/admin-frontend#16]
I think there are some missing values in the config example from the Readme (app.json in deployment repo) that prevent Admin Frontend from running on the server: 

Specifically: 
- "additionalImageSources": [],
- "additionalXHRImageSources": [],

**Comment by lukashroch** [comment]
@cheburan have you actually tried this? What error does it produce? ... checking @mechkg code (https://github.com/MRC-Epid-it24/admin-frontend/commit/3b11da60c49fc38fa678efa936520f2864b47a2a) it defaults to `[]` so build should work.

Though we should probably add them anyway to have the config complete.

**Comment by cheburan** [comment]
Hi @lukashroch. I've added them manually after I got an error message regarding the Missing values during the build process. 
```
fatal: [192.168.56.10]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'additionalImageSources'"}
```
this happens if the config doesn't have 
```
    "additionalImageSources": [],
    "additionalXHRImageSources": []
```

Checked that deployment repo is up to date.  And as I understand, the admin-frontend should've pulled latest version of code from git anyway,  before the build on the server.

**Comment by lukashroch** [comment]
Right, this one actually comes from ansible / deployment scripts. So if you update your instance config like this example (https://github.com/MRC-Epid-it24/deployment/commit/1e8fc0af474248d99cc45da39b2dde2589e1b948) it should work.
